### PR TITLE
Yuri branch

### PR DESCRIPTION
**Pull request by yuriarg**

### CODE REVIEW

File: domain_test.py [hunk 1]
```diff
@@ -9,6 +9,10 @@ def test_to_unicode(self):
         actual = domain.to_unicode()
         expected = Domain('jana-sováková.seria-z.net.')
         self.assertEqual(expected, actual)
+        domain = Domain('xn--frasesclebres-hhb.com.')
```
Comment (rozhn0v) [inline domain_test.py:12]: it's better to create a new test method for each case and to desribe the case in method's name:
test_<tested_method_name>_when_<some_short_description>

File: .pylint.cfg [hunk 2]
```diff
@@ -62,7 +62,8 @@ confidence=
 # --disable=W".
 disable=missing-module-docstring,
         missing-function-docstring,
-        missing-class-docstring
+        missing-class-docstring,
+        no-else-return
```
[the remark this answers is not in the document]
Comment (yuriarg) [reply .pylint.cfg:66]: Yes, you are right, I'm gonna revert to the previous version.

File: domain.py [hunk 3]
```diff
@@ -153,28 +155,43 @@ def correct_accent_equal(self, homo_domain: Domain) -> bool:
             try:
                 translation = domain_blob.translate(
                     from_lang=domain_blob.detect_language(),
-                    to=homo_blob.detect_language())
+                    to=homo_blob.detect_language()).replace(' ', '')
```
[the remark this answers is not in the document]
Comment (yuriarg) [reply domain.py:158]: I'm gonna turn it into a function later, but I sill didn't get the behavior that I want for this part, I'm gonna try to figure it out first.

File: domain.py [hunk 4]
```diff
@@ -153,28 +155,43 @@ def correct_accent_equal(self, homo_domain: Domain) -> bool:
...
+                if translation == homo_blob \
+                        or norm_domain == norm_translation:
+                    not_equivalent -= 1
                     continue
-                not_equivalent = 1
-        return not not_equivalent
+                not_equivalent += 1
+        if not_equivalent >= 0:
+            return -1
+        else:
```
[the remark this answers is not in the document]
Comment (yuriarg) [reply domain.py:193]: Yes, sometimes I make that mistake, it's redundant, that's one of the reasons the no-else-return option is important, thanks to point it out.

File: domain_test.py (hunk 1 above)
Comment (yuriarg) [reply domain_test.py:12]: I see, I'm gonna do it.

**Pull request merged**